core/App/ehsm_napi.cpp

```diff
@@ -84,35 +83,50 @@ void NAPI_Finalize(){
     Finalize();
 }
 
-/*
-@return
-[string] json string
+/**
+ * @brief Create key and save the parameters when using the key for encrypt, decrypt, sign and verify
+ * 
+ * @param paramJson Pass in the key parameter in the form of JSON string
+ * @return char* 
+ * [string] json string
     {
         code: int,
         message: string,
         result: {
-            cmk_base64 : string,
+            cmk_base64 : string
         }
     }
-*/
-char* NAPI_CreateKey(const uint32_t keyspec, const uint32_t origin)
+ */
```

Comment:
please add more description for each parameter, or i don't know I can get what parameters in the json string.

same with each below APIs.